server.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const cors = require('cors');
+require('dotenv').config();
+const PORT = process.env.PORT || 3003;
+const weatherData = require('./data/weather.json');
+
+app.get('/', (req, res) => {
+  res.send('Greeting check for root URL');
+});
+
+app.get('/weather', (req, res) => {
+  let city = req.query;
+  res.send(weatherData.city.city_name);
+});
```

Comment:
Good work so far  on task 1 and starting on task 2 for weather 👍 

- This endpoint will process the request that contains `lat, lon and the searchQuery`
- We will need to use `find()` to discover which city the `lat, lon and searchQuery` info belong to